x/kavamint/abci.go

```diff
@@ -11,9 +11,12 @@ func BeginBlocker(ctx sdk.Context, k keeper.Keeper) {
 	if !found {
 		previousBlockTime = ctx.BlockTime()
 	}
+	// calculate totals before any minting is done to prevent new mints affecting the values
```

Comment:
does the ordering of the keepers in app.go ensure that x/mint's begin blocker is called before the staking modules?

Reply:
yes, the ordering of the begin blockers matters. the kavamint's begin blocker must be registered before distribution so that the fee pool has the newly minted tokens to distribute. indicated the requirement in a comment where the begin blocker is registered.

https://github.com/Kava-Labs/kava/blob/18c41c2ef15fa59af9a8d363302dd8ad03b73747/app/app.go#L783-L785